library/auth.inc

```diff
@@ -85,7 +85,7 @@ if (
     // If session has timed out / been destroyed, logout record for null user/provider will be invalid.
     if (!empty($_SESSION['authUser']) && !empty($_SESSION['authProvider'])) {
         if ((isset($_GET['timeout'])) && ($_GET['timeout'] == "1")) {
-            EventAuditLogger::instance()->newEvent("timeout", $_SESSION['authUser'], $_SESSION['authProvider'], 1, "success");
+            EventAuditLogger::instance()->newEvent("logout", $_SESSION['authUser'], $_SESSION['authProvider'], 0, "timeout, so force logout");
```

Comment:
![smaller](https://user-images.githubusercontent.com/5618372/120527865-d75c3b00-c3a8-11eb-9238-aa6a0ded7649.gif)

Reply:
duh, just realized we already had a 'timeout' log entry message below in this same script :)
So, just made it the same :)